extensions/assets-manager/AssetsManagerEx.cpp

```diff
@@ -49,6 +49,10 @@ NS_CC_EXT_BEGIN
 const std::string AssetsManagerEx::VERSION_ID = "@version";
 const std::string AssetsManagerEx::MANIFEST_ID = "@manifest";
 
+bool SortCompressFileIndexAssert(const CompressedFilesInfoAsserts& a, const CompressedFilesInfoAsserts& b)
```

Comment:
it is under  namespace cocos2d { namespace extension { 
is it necessary?  conflict it not possible.
other purpose